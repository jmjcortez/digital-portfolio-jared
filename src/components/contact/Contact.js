import React from 'react';

import './Contact.styles.css';

import email from '../../assets/images/email.png';
import ig from '../../assets/images/ig.png';


const Contact = () => {
  return (
    <div className='contact-container' data-testid='contact-container'>
      <h1 className='title'>Thank You!!!</h1>
      <div className='contact-content-container'>
        <div className='text-container'>
          <img src={email} alt='heart' className='logo' />
          <span>Contact Jared through his Instagram or email jmjcortez@gmail.com for commisions or prints</span>
        </div>
        <div className='text-container'>
          <img src={ig} alt='ig' className='logo' />
          <span>Follow Jared on his social media accounts</span>
          <span>Instagram: <a className='ig-link' href='https://www.instagram.com/jared.m.cortez'>@jared.m.cortez</a></span>
        </div>
      </div>
    </div>
  )
}

export default Contact