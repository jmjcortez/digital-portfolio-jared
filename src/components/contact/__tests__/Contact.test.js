import '@testing-library/jest-dom';

import { render, screen, cleanup } from "@testing-library/react";
import renderer from 'react-test-renderer';

import Contact from "../Contact";

afterEach(() => {
  cleanup();
});

test('should render Contact', () => {
  render(<Contact />);
  const contactElement = screen.getByTestId('contact-container');
  expect(contactElement).toBeInTheDocument();
});

test('matches snaphshot', () => {
  const tree = renderer.create(<Contact />);
  expect(tree).toMatchSnapshot();
});