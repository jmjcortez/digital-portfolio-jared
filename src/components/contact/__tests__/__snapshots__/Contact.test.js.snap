// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snaphshot 1`] = `
<div
  className="contact-container"
  data-testid="contact-container"
>
  <h1
    className="title"
  >
    Thank You!!!
  </h1>
  <div
    className="contact-content-container"
  >
    <div
      className="text-container"
    >
      <img
        alt="heart"
        className="logo"
        src="email.png"
      />
      <span>
        Contact Jared through his Instagram or email jmjcortez@gmail.com for commisions or prints
      </span>
    </div>
    <div
      className="text-container"
    >
      <img
        alt="ig"
        className="logo"
        src="ig.png"
      />
      <span>
        Follow Jared on his social media accounts
      </span>
      <span>
        Instagram: 
        <a
          className="ig-link"
          href="https://www.instagram.com/jared.m.cortez"
        >
          @jared.m.cortez
        </a>
      </span>
    </div>
  </div>
</div>
`;
